README.md

```diff
@@ -63,11 +63,7 @@ One can open or check existing issues on [GitHub issues](https://github.com/bale
 
 ## Deploying
 
-After a PR has been created, a member of the balena team will review and merge the PR. The pull request will undergo CI/CD checks to verify if everything is correct. For example: if a Change-type has been added. More information present in the [CONTRIBUTING document](CONTRIBUTING.md)
```

Comment:
I removed this because I thought it wasn't necessary to explain that we will review your PR and verify everything works.